test/vcr_setup.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'vcr'
+# VCR records HTTP interactions to cassettes that can be replayed during unit tests
+# allowing for faster, more predictible web interactions
+VCR.configure do |c|
+  # This is where the various cassettes will be recorded to
+  c.cassette_library_dir = 'test/vcr_cassettes'
+  # This permits non-VCR webmock requests in other tests
+  c.allow_http_connections_when_no_cassette = true
+  c.hook_into :webmock
+
+  # To avoid storing plain text Twilio API keys or requiring the keys
+  # be provided at every run of the rake tests, provide the following credentials
+  # whenever you need to record a cassette that requires valid credentials
+  # these should be stored in your local_env.yml file anyhow
+  ENV['TWILLIO_STUDIO_FLOW'] = 'test_studio_flow' unless ENV['TWILLIO_STUDIO_FLOW']
```

Comment:
could these be simplified to `ENV['TWILLIO_STUDIO_FLOW'] ||= 'test_studio_flow'`

Reply:
This ended up causing errors so I reverted to the previous code that uses `unless`